polaris-for-figma/package.json

```
@@ -1,6 +1,7 @@
 {
   "name": "polaris-for-figma",
```

```suggestion
  "name": "@shopify/polaris-for-figma",
```

Let's stick this in the shopify namespace so in the event it happens again, at least it is part of the org and org admins can unpublish it rather than having to resort to gh actions shennanigans.